button-examples/Web/web-button-example.md

```diff
@@ -9,6 +9,8 @@ There are four types of buttons:
 3. [Contained button](#contained-button)
 4. [Toggle button](#toggle-button)
 
+TODO: Remove Toggle button section (or rename it to **Icons as button**), Web does not have Toggle button as described in design guidance.
+
```

Comment:
We will retain the toggle button as a stub to remain consistent with the guidance.